freud/order.pyx

```diff
@@ -175,10 +175,24 @@ cdef class Nematic(_Compute):
     def compute(self, orientations):
         R"""Calculates the per-particle and global order parameter.
 
+        Example::
+
+            >>> # Assuming user has set director = np.array[x,y,z]
+            >>> # Assuming box, points/positions, and system (box, points)
+            >>> # have already been defined
+            >>> # Recommend visualizing the data with the particle_property below
+            >>> import freud
+            >>> from ovito.data import *
+            >>> import numpy as np
+            >>> order_param = freud.order.Nematic(director)
+            >>> order_param.compute(points.orientations)
```

Comment:
`points` should be an `Nx3` array in these examples so they should not have an `orientation` attribute. You may want to assign fake orientations in this example or include this in your comments as a preassigned value.